index.html

```diff
@@ -647,7 +647,7 @@ <h3 class="pivx-bold-title center-text"><span>Control your</span> Masternode</h3
          
             <div id='mnTextErrors' style='display: block;'>
               <br>
-              <p class="center-text">Please access your <b>COLLATERAL WALLET </b> first.</p>
+              <p class="center-text"></p>
```

Comment:
Oh, looks like domMnTextErrors always accesses `.children[1]`, which is this empty `<p>`. It should point to the `<p>` instead of the div

Reply:
I left it empty intentionally (as that text is now set dynamically by the MN display code).

I wasn't sure why `domMnTextErrors` was structured like that, but I left it as-is in case it was somehow necessary, could certainly be simplified down to a single element, shall I tackle this later?

Reply:
Already did it myself :)

Reply:
:purple_heart: